client/web/src/components/panels/Document.vue

```diff
@@ -319,9 +315,6 @@ export default defineComponent({
 
 		window.addEventListener("keyup", (e: KeyboardEvent) => this.keyUp(e));
 		window.addEventListener("keydown", (e: KeyboardEvent) => this.keyDown(e));
-
-		// TODO: Implement an actual UI for chosing colors (this is completely temporary)
-		this.updatePrimaryColor({ r: 247 / 255, g: 76 / 255, b: 0 / 255, a: 0.6 });
 	},
```

Comment:
```scss
.viewport {
			flex: 1 1 100%;
			.canvas {
				background: var(--color-1-nearblack);
				width: 100%;
				height: 100%;
				svg {
					width: 100%;
					height: 100%;
				}
```

We can add `background: #ffffff;` in order to make the SVG canvas white so our default black primary color shapes will be visible.